verdaccio/load-tests/scripts/load-test.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Load test with the purpose of to determine how Verdaccio responses under an average load.
+ *
+ * Ramps up the number of users over 1 minute, then keeps the load stable for 20 mins,
+ * and then ramps down over 1 min.
```

Comment:
Comment doesn't match actual numbers. Probably better to drop the comment altogether.